features/tests/features/generators/test_category.py

```diff
@@ -48,22 +48,22 @@ def test_category_feature_generator(generator_helper, data_helper):
     ]
 
     # When
-    output_datas = []
+    output_data = []
```

Comment:
`output_data` is already a variable being used in this test, so it cannot be used as this variables name.